packages/bundler-security/src/constants.js

```diff
@@ -0,0 +1,12 @@
+const TREZOR_PACKAGES_SCOPES = ['@trezor', '@suite-common', '@suite-native'];
+const PACKAGES_PATHS = ['/packages/', '/suite-common/', '/suite-native/'];
+
+const RELATIVE_PATH_REGEX = new RegExp(
+    `(?:\\.\\./){2,}(?:${PACKAGES_PATHS.map(p => p.slice(1, -1)).join('|')})`,
+);
+
+module.exports = {
+    TREZOR_PACKAGES_SCOPES,
+    PACKAGES_PATHS,
```

Comment:
PACKAGES_PATHS are not imported anywhere, does it make sense to export it?

Reply:
i think it's fine to remove it